Networking/Networking/Mapper/TaxClassMapper.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+
+/// Mapper: TaxClass
+///
+struct TaxClassMapper: Mapper {
```

Comment:
is this struct used in a followup task? it doesn't look like we can fetch just one tax class from the API doc: https://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-tax-classes (only "load all", "create" and "delete")